.changeset/itchy-hats-ring.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-pwa/helpers-next": minor
+---
+
+Added `getCategoryRoute` and `getProductRoute` helpers to create router lins with history API metadata
```

Comment:
How does it work with router hash mode if someone would need to change it (donno why)? does it have anything to do with the implementation?

Reply:
Haven't tested it but it shouldn't affect it at all, but then someone probably will build links in another way. The mechanism should work though

Reply:
ok it's fine, it's just a template with proposed implementation. We will see if someone faces a problem with that or not 👍🏻